Examples/TicTacToe/Sources/Core/LoginCore.swift

```diff
@@ -38,63 +38,63 @@ public struct LoginEnvironment {
   }
 }
 
-public let loginReducer =
+public let loginReducer = Reducer<LoginState, LoginAction, LoginEnvironment>.combine(
   twoFactorReducer
-  .optional()
```

Comment:
this is mostly just a re-indentation so best to view this PR with whitespace off.